web_src/css/modules/label.css

```diff
@@ -0,0 +1,571 @@
+}
+.ui.orange.labels .label,
+.ui.ui.ui.orange.label {
+  background: var(--color-orange);
+  border-color: var(--color-orange);
+  color: var(--color-white);
+}
+
+.ui.orange.labels a.label:hover,
+a.ui.ui.ui.orange.label:hover {
+  background: var(--color-orange-dark-1);
+  border-color: var(--color-orange-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .orange.label,
+.ui.ui.ui.basic.orange.label {
+  background: transparent;
+  border-color: var(--color-orange);
+  color: var(--color-orange);
+}
+.ui.basic.labels a.orange.label:hover,
+a.ui.ui.ui.basic.orange.label:hover {
+  background: transparent;
+  border-color: var(--color-orange-dark-1);
+  color: var(--color-orange-dark-1);
+}
+.ui.yellow.labels .label,
+.ui.ui.ui.yellow.label {
+  background: var(--color-yellow);
+  border-color: var(--color-yellow);
+  color: var(--color-white);
+}
+
+.ui.yellow.labels a.label:hover,
+a.ui.ui.ui.yellow.label:hover {
+  background: var(--color-yellow-dark-1);
+  border-color: var(--color-yellow-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .yellow.label,
+.ui.ui.ui.basic.yellow.label {
+  background: transparent;
+  border-color: var(--color-yellow);
+  color: var(--color-yellow);
+}
+.ui.basic.labels a.yellow.label:hover,
+a.ui.ui.ui.basic.yellow.label:hover {
+  background: transparent;
+  border-color: var(--color-yellow-dark-1);
+  color: var(--color-yellow-dark-1);
+}
+.ui.olive.labels .label,
+.ui.ui.ui.olive.label {
+  background: var(--color-olive);
+  border-color: var(--color-olive);
+  color: var(--color-white);
+}
+
+.ui.olive.labels a.label:hover,
+a.ui.ui.ui.olive.label:hover {
+  background: var(--color-olive-dark-1);
+  border-color: var(--color-olive-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .olive.label,
+.ui.ui.ui.basic.olive.label {
+  background: transparent;
+  border-color: var(--color-olive);
+  color: var(--color-olive);
+}
+.ui.basic.labels a.olive.label:hover,
+a.ui.ui.ui.basic.olive.label:hover {
+  background: transparent;
+  border-color: var(--color-olive-dark-1);
+  color: var(--color-olive-dark-1);
+}
+.ui.green.labels .label,
+.ui.ui.ui.green.label {
+  background: var(--color-green);
+  border-color: var(--color-green);
+  color: var(--color-white);
+}
+
+.ui.green.labels a.label:hover,
+a.ui.ui.ui.green.label:hover {
+  background: var(--color-green-dark-1);
+  border-color: var(--color-green-dark-1);
+  color: var(--color-white);
+}
+
+.ui.basic.labels .green.label,
+.ui.ui.ui.basic.green.label {
+  background: transparent;
+  border-color: var(--color-green);
+  color: var(--color-green);
+}
+.ui.basic.labels a.green.label:hover,
+a.ui.ui.ui.basic.green.label:hover {
+  background: transparent;
+  border-color: var(--color-green-dark-1);
+  color: var(--color-green-dark-1);
+}
+.ui.teal.labels .label,
+.ui.ui.ui.teal.label {
+  background: var(--color-teal);
+  border-color: var(--color-teal);
+  color: var(--color-white);
+}
+
+.ui.teal.labels a.label:hover,
```

Comment:
We really need to get rid of these custom colors like `teal` or `orange`.
There are currently perhaps two mentions of this class.
Can't we use one of the existing colors for it?

Reply:
Indeed. I think red,green and maybe blue is all that's needed. Maybe orange to indicate a warning status. But others can likely remove.